addon/components/bs-accordion/item/title.hbs

```diff
@@ -17,24 +17,15 @@
   </h2>
 {{else}}
   <div
-    class="{{if (macroCondition (macroGetOwnConfig "isBS3")) "panel-heading"}} {{if (macroCondition (macroGetOwnConfig "isBS4")) "card-header"}}"
+    class="{{if (macroCondition (macroGetOwnConfig "isBS4")) "card-header"}}"
```

Comment:
`macroCondition` isn't needed here anymore. The code is only run for BS4

Reply:
Nice catch, fixed! :+1: